src/main/java/io/github/redouane59/twitter/ITwitterClientV2.java

```diff
@@ -289,6 +318,25 @@
    */
   TweetList getUserTimeline(String userId, AdditionalParameters additionalParameters);
 
+  /**
+   * Get the most recent Tweets posted by the user calling https://api.twitter.com/2/users/:id/tweets (time & tweet id arguments can be null)
+   *
+   * @param userId identifier of the Twitter account (user ID) for whom to return results.
+   * @param requestTweetFieldsScope 
+   * @return a TweetList object containing a list of tweets and the next token if recursiveCall is set to false
+   */
+  public TweetList getUserTimeline(final String userId, REQUEST_TWEET_FIELDS_SCOPE requestTweetFieldsScope);
+  
+  /**
+   * Get the most recent Tweets posted by the user calling https://api.twitter.com/2/users/:id/tweets (time & tweet id arguments can be null)
+   *
+   * @param userId identifier of the Twitter account (user ID) for whom to return results.
+   * @param additionalParameters accepted parameters recursiveCall, startTime, endTime, sinceId, untilId, maxResults
+   * @param requestTweetFieldsScope
+   * @return a TweetList object containing a list of tweets and the next token if recursiveCall is set to false
+   */
+  public TweetList getUserTimeline(String userId, AdditionalParameters additionalParameters, REQUEST_TWEET_FIELDS_SCOPE requestTweetFieldsScope);
```

Comment:
Little question, isn't it possible to add the `requestTweetFieldsScope` parameter inside the `additionalParameters` object?
It will avoid creating new constructors no ?